modules/android/examples/codesnippet_collection.java

```diff
@@ -628,25 +626,16 @@ public void oneXAttachmentsExample(Database database) {
     public void newDatabaseExample() throws CouchbaseLiteException {
         final String customDir = "/foo/bar";
         // tag::new-database[]
-        DatabaseConfiguration config = new DatabaseConfiguration();
-        config.setDirectory(customDir);
-        Database database = new Database(DB_NAME, config);
+        Database database = new Database(DB_NAME);
         // end::new-database[]
-        
+
         // tag::close-database[]
         database.close();
         // end::close-database[]
 
         database.delete();
     }
 
-     public void databaseFullSyncExample() throws CouchbaseLiteException {
```

Comment:
Has this been redone somewhere else? This snippet is required for 3.1.9 onwards so can this be added back in please?